web/styles/settings.css

```diff
@@ -1634,6 +1634,8 @@ $option_title_width: 180px;
 
 .settings-field-label {
     margin-bottom: var(--margin-bottom-field-description);
+    /* Avoid having the clickable area extend to the full width of the containing element */
```

Comment:
I guess we should also have this change for `modal-field-label`.

Reply:
Also, the label for stream posting policy setting does not use the `setting-field-label` class. So, we may also need to audit all the label elements once.

Reply:
Yeah I've set it for: 

- modal-field-label
- setting-field-label
- group-setting-label

Some labels in stream posting policy were missing its class name so I've added the `setting-field-label` class to them.